packages/fxa-payments-server/src/routes/Subscriptions/index.scss

```diff
@@ -23,6 +23,10 @@
     margin: 0 -16px;
   }
 
+  hr + hr {
+    border: 0;
```

Comment:
Do our horizontal rules take up any vertical space (like margin or padding)? If so we may be better off setting `display: none`, otherwise it'll be visually hidden while still taking up space. If not, nvm!

Reply:
In this case we want it to take up space

Reply:
I dont want to block you on this train, but I’d probably recommend reworking some of the styles to add the additional space that’s desired here, instead of doubling up on HRs. Can be a follow up or just a mental note. It’s also possible I’m missing some context here. Carry on!

Reply:
HRs for spacing is not the most semantic, I agree. Just wanted to get this quick fix out and it was already doubled up HRs, I will file a follow up